src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/ListViewTests.cs

```diff
@@ -0,0 +1,521 @@
+        {
+            var columnHeader1 = new ColumnHeader { Text = "ColumnHeader1", Width = 140 };
+            var columnHeader2 = new ColumnHeader { Text = "ColumnHeader2", Width = 140 };
+            var columnHeader3 = new ColumnHeader { Text = "ColumnHeader3", Width = 140 };
+            listView.Columns.AddRange(new[] { columnHeader1, columnHeader2, columnHeader3 });
+            var listViewItem1 = new ListViewItem(new[] { "row1", "row1Col2", "row1Col3" }, -1) { StateImageIndex = 0 };
+            var listViewItem2 = new ListViewItem(new[] { "row2", "row2Col2", "row2Col3" }, -1) { StateImageIndex = 0 };
+            var listViewItem3 = new ListViewItem(new[] { "row3", "row3Col2", "row3Col3" }, -1) { StateImageIndex = 0 };
+            listView.RetrieveVirtualItem += (s, e) =>
+            {
+                e.Item = e.ItemIndex switch
+                {
+                    0 => listViewItem1,
+                    1 => listViewItem2,
+                    2 => listViewItem3,
+                    _ => listViewItem1,
+                };
+            };
+
+            listView.CheckBoxes = checkBoxesEnabled;
+            listView.FullRowSelect = true;
+            listView.View = view;
+            listView.VirtualMode = virtualModeEnabled;
+            listView.VirtualListSize = 3;
+            if (!virtualModeEnabled)
+            {
+                listView.Items.AddRange(new[] { listViewItem1, listViewItem2, listViewItem3 });
+            }
+        }
+
+        private void InitializeTileList(ListView listView, int columnCount, int subItemsCount, Size tileSize)
+        {
+            listView.View = View.Tile;
+            listView.TileSize = tileSize;
+
+            for (int i = 0; i < columnCount; i++)
+            {
+                listView.Columns.Add(new ColumnHeader() { Text = $"ColumnHeader{i}" });
+            }
+
+            ListViewItem listViewItem = new ListViewItem("Test");
+            for (int i = 0; i < subItemsCount; i++)
+            {
+                listViewItem.SubItems.Add(new ListViewItem.ListViewSubItem() { Text = $"Test SubItem{i}" });
+            }
+
+            listView.Items.Add(listViewItem);
+        }
+
+        public static Point GetCenter(Rectangle rect)
+        {
+            int x = rect.Left + ((rect.Right - rect.Left) / 2);
+            int y = rect.Top + ((rect.Bottom - rect.Top) / 2);
+            return new Point(x, y);
+        }
+
+        private async Task RunTestAsync(Func<Form, ListView, Task> runTest)
+        {
+            await RunSingleControlTestAsync(
+                testDriverAsync: runTest,
+                createControl: () =>
+                {
+                    ListView control = new()
+                    {
+                        Size = new Size(400, 100)
+                    };
+
+                    return control;
+                },
+                createForm: () =>
+                {
+                    return new();
+                });
```

Comment:
We probably want to be more explicit about the size of the form. E.g., what would happen if the test form was somehow narrower or shorter than (400, 100)? 
```suggestion
                createControl: () =>
                {
                    ListView control = new()
                    {
                       Dock = DockStyle.Fill
                    };

                    return control;
                },
                createForm: () =>
                {
                    return new()
                    {
                        Size = new Size(500, 400)
                    };
                });
```